erigon-lib/diagnostics/sys_info.go

```diff
@@ -134,27 +134,21 @@ func GetDiskInfo(nodeDisk string) DiskInfo {
 	}
 }
 
-func GetCPUInfo() CPUInfo {
-	modelName := ""
-	cores := 0
-	mhz := float64(0)
+func GetCPUInfo() []CPUInfo {
+	cpuinfo := make([]CPUInfo, 0)
 
 	cpuInfo, err := cpu.Info()
 	if err == nil {
 		for _, info := range cpuInfo {
-			modelName = info.ModelName
-			cores = int(info.Cores)
-			mhz = info.Mhz
-
-			break
+			cpuinfo = append(cpuinfo, CPUInfo{
+				ModelName: info.ModelName,
+				Cores:     info.Cores,
```

Comment:
Better add method Clone/Copy to object
Also maybe in this case can use slices.Clone() - because no pointers inside objects